<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartWindow</class>
 <widget class="QWidget" name="ChartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Efficient Fronteir</string>
  </property>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
